django/db/backends/mysql/schema.py

```diff
@@ -54,10 +54,22 @@ def skip_default(self, field):
 
     @property
     def _supports_limited_data_type_defaults(self):
-        # Only MariaDB >= 10.2.1 supports defaults for BLOB and TEXT.
+        # MariaDB >= 10.2.1 and MySQL >= 8.0.13 supports defaults for BLOB
+        # and TEXT.
         if self.connection.mysql_is_mariadb:
             return self.connection.mysql_version >= (10, 2, 1)
-        return False
+        return self.connection.mysql_version >= (8, 0, 13)
```

Comment:
For MySQL >= 8.0.13, `default` was not fully supported as other backends, for example:
Alter a field with default value:
✅ `ALTER TABLE foo MODIFY COLUMN bar LONGTEXT DEFAULT("");
❌ `ALTER TABLE foo ALTER COLUMN bar SET DEFAULT ('');
So unless we could tell what kind of action of this SQL is taken, otherwise we should always return `False` for safe.

Reply:
see ticket-32503